docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    image: todoapp:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: todoapp
+    ports:
+      - "8080:8080"
+    environment:
+      - ENV=local
+    networks:
+      - db-data-net
+    depends_on:
+      mysql-local:
+        condition: service_started
+    restart: unless-stopped
+  mysql-local:
+    image: mysql:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: mysql-local
+    ports:
+      - "3306:3306"
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_DATABASE=app_db
+      - MYSQL_USER=app_user
+      - MYSQL_PASSWORD=1234
```

Comment:
This envs is set up on the Dockerfile of MySQL, no need to define it one mire time